frontstage/controllers/party_controller.py

```diff
@@ -88,6 +88,21 @@ def create_account(registration_data):
     logger.info('Successfully created account')
 
 
+def update_account(respondent_data):
```

Comment:
There should be a simple unit test for this function.  Doesn't need to be much, but what does it return when you get a 200 or 404?  Looking at it, it just returns text so I think thats useful to highlight via a unit test

Reply:
its now a part of the integration test which suggests that.